### ISSUE TITLE
[Epic] Make drill-through chiller

### ISSUE THREAD

**Issue by cdeweyx** [metabase/metabase#26836]
[Product doc](https://www.notion.so/metabase/Make-drill-through-chiller-0caf3f254da0463dbe7931f086f5ee28)

**Comment by ranquild** [comment]
@deniskaber yep, the last about `contains/does not contain` filter is exactly that
